tests/stress_test/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "eos-stress-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "stress_test.js",
+  "dependencies": {
+    "lodash": "~4.17.4",
+    "request": "~2.83.0",
+    "request-promise": "~4.2.2"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://bitbucket.org/ii5/stress-test.git"
```

Comment:
Should this be eos github?

Reply:
Ah yes, good catch. It's because this stress test script is part of another repo before I moved it here. I'm going to update it.